front/internals/jest/setup.js

```diff
@@ -31,3 +31,12 @@ jest.mock('history', () => ({
     parsePath: jest.fn(),
   }),
 }));
+
+jest.mock('utils/cl-router/Link');
+jest.mock('utils/cl-router/history');
+jest.mock('hooks/useLocale');
+jest.mock('hooks/useLocalize');
+jest.mock('utils/cl-intl');
+jest.mock('services/locale');
+jest.mock('services/appConfiguration');
+jest.mock('modules', () => ({ streamsToReset: [] }))
```

Comment:
Modules seems to be mocked already above.

Reply:
Ah, good catch!